### ISSUE TITLE
Review for 2015 Brainhack Proceedings

### ISSUE THREAD

**Issue by tyarkoni** [Brainhack-Proceedings-2015/Ding-OHBM-niftyview#1]
I've carefully read the Markdown doc summarizing the project, and I think it looks good overall. My main recommendation would be to add a few sentences comparing NiftyView to other web-based viewers. You mention that NiftyView offers advantages over desktop viewers, but there are a number of other online viewers out there (most notably [Papaya](https://github.com/rii-mango/Papaya), which is used in [NeuroVault](http://neurovault.org), but also, e.g., [BrainBrowser](https://brainbrowser.cbrain.mcgill.ca/) and [slicedrop.com](http://slicedrop.com)). It would be good to briefly mention any advantages or disadvantages of NiftyView compared to these other services.

There could also perhaps be slightly more discussion of:
- The current limitations of NiftyView compared to desktop viewers (e.g., does NiftyView support overlays of multiple images?)
- Any current restrictions on image format. E.g., it appears that NiftyView currently doesn't support gzipped nifti images (I tried to drop a few on the website, and nothing happened; it worked fine when I decompressed them first).

None of these are major issues and I don't think it should take more than a couple of sentences to address each of them.

**Comment by weirandeng** [comment]
We appreciate the reviewer's constructive comments and have made corresponding changes in the revision. 

Comparing to existing web-based image viewers such as Papaya, BrainBrowser, and Slice:Drop, which are more mature, NifityView has very limited features. We strongly agree with the reviewers that there is little significance in developing yet another web-based image viewer without comparing and discussing the difference between the NiftyView and other viewers. In Discussion, we included in our future plan that more features will be developed and released. Also, we updated with our current development status. Our primary goal is to develop a really fast technology for displaying and processing medical images in web browser. A preliminary result of a factor of six to eight acceleration using WebGL for accelerating image processing has been presented in the revision. We plan to develop more WebGL-accelerated image processing features in future development.
